### PR TITLE
(GH-33) Draft rules for Taunts

### LINKED ISSUE #33: Draft Taunts Rules
Write a section on taunts explaining their general structure/how to invoke them with a list of generic taunts any group could conceivably invoke (plus some that are limited by group profile/traits).

For reference, see [the blog post](https://flagrant.garden/posts/taunts/).

### PR DESCRIPTION
**Pull request by michaeltlombardi**
This PR adds Taunts to the base game, triggered via the new Proclaim order. Taunts change the way the proclaiming Group plays for the duration of the skirmish.

Resolves #33.

See [the Rules page](https://deploy-preview-37--flagrant-garden.netlify.app/games/factions/rules/) for the rules rendered on the site.

### CODE REVIEW

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:253]:
```suggestion
| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS instead if they would be forced to do so), and all enemy Groups are Reckless against them.                           |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:396]:
```suggestion
A Group may activate to Proclaim a taunt, potentially introducing a new special rule affecting them for the remainder of the skirmish.
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:397]:
```suggestion
To Proclaim, the Group must activate with roll that meets or exceeds the Check value for the specific Taunt they wish to Proclaim.
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:256]:
```suggestion
| We Never Miss a Shot!          |  6+   | Groups with an MI profile only.                  | When this Group activates to Shoot, they must Shoot the farthest enemy Group in range (if any) but may reroll any misses when Shooting.                                     |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:258]:
```suggestion
| You'll Die with Us!            |  6+   | Must be Bloodied                                 | When Attacked, this Group inflicts one hit against their attackers for each hit they take, but have -1 T.                                                                           |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:259]:
```suggestion
| There Can Be Only One!         |  7+   | -                                                | Pick an enemy Group. This Group must activate to Attack or Move toward that Group. When Attacking the chosen Group, gain +1 T, and treat this Group's MO as double its written value. |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:260]:
```suggestion
| We'll Fight in the Shade!      |  7+   | -                                                | This Group has +1 T when an enemy Shoots them, but gain the Reckless trait against enemy Groups with an MI profile.                                   |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:257]:
```suggestion
| We'll Never Taste Your Blades! |  6+   | -                                                | +1 T and +1 on to-hit rolls when Attacking, but hits against this Group always count twice.                                              |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:261]:
```suggestion
| You'll Never Take Us Alive!    |  7+   | -                                                | Cannot Retreat (-1 FS instead if they would be forced to do so), but +1 on to-hit rolls when Attacking or being Attacked.                       |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:262]:
```suggestion
| For the Captain!               |  8+   | Must be within 12" of Captain.                    | While within 12" of Captain, this Group rerolls all failed Activations. Otherwise, reroll all successful activations and Resolve tests.            |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:263]:
```suggestion
| We'll Run You Down!            |  8+   | Groups with an MO of 8" or more only.           | This Group gains the Reckless trait, +2" MO, and may only activate to Attack or Move.                                           |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:264]:
```suggestion
| Your Magic is Laughable!       |  8+   | Groups with the Apprentice or Caster traits only. | This Group may force an enemy to reroll a Cast activation against them. Any spell which still succeeds is twice as effective.                        |
```

Comment (s-jared) [reply content/games/factions/rules.md:264]: Heads up: this is a functional change ("against them" to limit use a bit). It seemed to need some direction, could have been "a Cast activation within 12"" or something instead. I think I like this one better.

**Review by s-jared — CHANGES_REQUESTED** [review]
Okay I've gone through the Taunt material. Let me know if anything looks funny! Still finding the corners of this interface, obviously.

File: content/games/factions/rules.md
Comment (michaeltlombardi) [reply content/games/factions/rules.md:253]: Should that comma be there or should it read:
> Cannot Move or Retreat (-1 FS instead if they would be forced to do so) and all enemy Groups are Reckless against them.
```suggestion
| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS instead if they would be forced to do so) and all enemy Groups are Reckless against them.                           |
```

File: content/games/factions/rules.md
Comment (michaeltlombardi) [reply content/games/factions/rules.md:259]: Is the second comma needed here, or should it be:
> When Attacking the chosen Group, gain +1 T and treat this Group's MO as double its written value.
I _think_ if we removed the conditional at the start we wouldn't have that second comma, right?
```suggestion
| There Can Be Only One! | 7+ | - | Pick an enemy Group. This Group must activate to Attack or Move toward that Group. When Attacking the chosen Group, gain +1 T and treat this Group's MO as double its written value. |
```

File: content/games/factions/rules.md
Comment (s-jared) [reply content/games/factions/rules.md:253]: either is fine actually. now im looking at it, the ethos of this doc probably leans toward your suggestion.

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:257]:
```suggestion
| We'll Never Taste Your Blades! |  6+   | -                                                 | When attacking, this Group has +1 T and +1 on to-hit rolls. Hits against this Group always count twice.                                                                                                      |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:258]:
```suggestion
| You'll Die with Us!            |  6+   | Must be Bloodied                                  | -1 T. When Attacked, this Group inflicts one hit against their attackers for each hit they take.                                                                                             |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:261]:
```suggestion
| You'll Never Take Us Alive!    |  7+   | -                                                 | This Group cannot Retreat (-1 FS instead if they would be forced to do so). They have +1 on to-hit rolls when Attacking or being Attacked.                                                         |
```

**Review by s-jared — DISMISSED** [review]
Okay see how this looks, now.

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:255]:
```suggestion
| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group.               | While within 6" of a friendly Group, this Group may reroll any dice. This Group loses 1d6 FS whenever a friendly Group within 6" Routs.                                                           |
```

Comment (s-jared) [inline content/games/factions/rules.md:255]:
```suggestion
| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group.               | While within 6" of a friendly Group, this Group may reroll any dice. This Group loses 1d6 FS whenever a friendly Group within 6" Routs.                                                           |
```

File: content/games/factions/rules.md
Comment (s-jared) [inline content/games/factions/rules.md:258]:
```suggestion
| You'll Die with Us!            |  6+   | Must be Bloodied.                                  | -1 T. When Attacked, this Group inflicts one hit against their attackers for each hit they take.                                                                                                  |
```

**Review by s-jared — CHANGES_REQUESTED** [review]
Found two missing periods, looks good otherwise. I've chosen the "request changes" option here because once those go in, I'd like to go ahead and merge. I *believe* that's what that feature does, but do disabuse me of that notion if I'm incorrect.

Also I used the "Start A Review" thing for this pass (of two comments lmao), so I'm taking this opportunity to praise myself. :)

**Comment by michaeltlombardi** [comment]
> Found two missing periods, looks good otherwise. I've chosen the "request changes" option here because once those go in, I'd like to go ahead and merge. I _believe_ that's what that feature does, but do disabuse me of that notion if I'm incorrect.
> 
> Also I used the "Start A Review" thing for this pass (of two comments lmao), so I'm taking this opportunity to praise myself. :)

Okay, @s-jared, I think this is ready to go now - applied your suggestions and cleaned the history/formatting again. So now when you come back into the interface and approve the PR, you'll still have to come down here and merge, but your approval should be what makes the `Merging is blocked` error go away. 😊

**Review by s-jared — APPROVED** [review]

**Pull request merged**
